files/zh-cn/web/html/element/title/index.md

```diff
@@ -75,16 +75,14 @@ slug: Web/HTML/Element/title
 
 ### 示例
 
-```plain
-<title>Menu - Blue House Chinese Food - FoodYum: Online takeout today!</title>
+```html
+<title>菜单 - 蓝屋中餐 - FoodYum：今日可送达的在线外卖！</title>
```

Comment:
Use the same format as the one on [Taobao](https://www.taobao.com/)

![image](https://github.com/mdn/translated-content/assets/15844309/86521286-fb51-456e-8db1-aada72091da7)